src/__tests__/FusionAuthProvider.test.tsx

```diff
@@ -28,21 +29,19 @@ describe('FusionAuthProvider', () => {
         jest.spyOn(window, 'location', 'get').mockReturnValue(mockedLocation);
 
         const wrapper = ({ children }) => (
-            <FusionAuthProvider
-                baseUrl="https://sandbox.fusionauth.io"
-                clientID="85a03867-dccf-4882-adde-1a79aeec50df"
-                scope="openid offline_access"
-                redirectUri="http://localhost"
-                idTokenHint=""
-            >
+            <FusionAuthProvider configuration={TEST_CONFIGURATION}>
                 {children}
             </FusionAuthProvider>
         );
         const { result } = renderHook(() => useFusionAuthContext(), {
             wrapper,
         });
 
-        await result.current.login('state');
+        await waitFor(() =>
+            act(() => {
+                result.current.login('state');
+            }),
+        );
```

Comment:
`waitFor` wraps in `act` under the hood so I don't think this nesting is necessary